src/apple.rs

```diff
@@ -43,26 +55,46 @@ pub struct AppleResponse {
     #[serde(rename = "is-retryable")]
     pub is_retryable: Option<bool>,
     pub environment: Option<String>,
+    /// The latest Base64 encoded app receipt. Only returned for receipts that contain auto-renewable subscriptions. 
+    #[serde(rename = "latest-receipt")]
+    pub latest_receipt: Option<String>,
     /// An array that contains all in-app purchase transactions. This excludes transactions for consumable products
     /// that have been marked as finished by your app. Only returned for receipts that contain auto-renewable subscriptions.
+    #[serde(rename = "latest-receipt-info")]
```

Comment:
Added the latest_receipt field just as a way of checking if this is a subscription. latest_receipt_info might work on its own, but the wording of "consumable products that have been marked as finished" has me second guessing whether this will include transactions that haven't been marked as finished?

Reply:
I think we will never have one receipt that contains both